include/Packet.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <cstdint>
+
+class Packet {
+public:
+    Packet(std::uint64_t sender_id, std::uint64_t sending_time_ns, std::uint64_t delivery_time_ns);
+    Packet() = default;
+    ~Packet() = default;
+
+    bool operator<(const Packet& other) const;
+
+    /**
+     * @brief ID of the sending network device
+     */
+    std::uint32_t sender_id;
+
+    /**
+     * @brief Packet sending time in nanoseconds
+     */
+    std::uint64_t sending_time_ns;
+
+    /**
+     * @brief Packet delivery time in nanoseconds
+     */
+    std::uint64_t estimated_delivery_time_ns;
```

Comment:
Could you, please, explain why is it *estimated* delivery time instead of just delivery time?

Reply:
Also, I wonder should this be in a packet header at all...?

Reply:
This value corresponds to the delivery time, I named it estimated because of a misunderstanding.